### ISSUE TITLE
[SF_ACC][1.7.1][CheckBox] JAWS does not announce the readonly state of CheckBox

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#5841]
### **Bug Description**
1. JAWS does not announce the readonly state of CheckBox.
2. JAWS should not announce "press spacebar to check" when the `readonly` property is true since nothing happens after press spacebar.

### **Expected Behavior**
1. The readonly state should be announced by screen reader.
2. Screen reader should not announce "press spacebar to check" when the `readonly` property is true.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/CheckBox/
2. Focus the checkbox which `readonly` property is true
3. JAWS does not announce the readonly state and it announces "press spacebar to check"

### **Isolated Example**
https://sap.github.io/ui5-webcomponents/playground/components/CheckBox/

### **Context**
- UI5 Web Components version: 1.7.1
- OS/Platform: Windows10
- Browser: Chrome
- Affected component: CheckBox
- Screen Reader: JAWS2022

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [ ] Medium
- [x] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {...}
- Business impact: {...}

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

Apparently there are two issues here regarding the `readonly` CheckBox:

1. The `readonly` attribute is not announced where the `checked` is.
2. The announcer reads 'press spacebar to uncheck' where nothing happens if Space is pressed.

Could you please look into it ?

Regards,
Plamen Ivanov

**Comment by dobrinyonkov** [comment]
Closing this as JAWS Bug.

#425 there is an opened issue for JAWS that the readonly state is not anonunced,
as well as an open discussion about the aria attribute #1309
